code/modules/mob/living/carbon/human/human_emote.dm

```diff
@@ -216,7 +216,20 @@
 	key_third_person = "salutes"
 	message = "salutes."
 	message_param = "salutes to %t."
+	emote_type = EMOTE_AUDIBLE
```

Comment:
I think this is more of what you meant to do. EMOTE_SOUND will mark it as an audio emote, and hide it from runechat. 

```suggestion
	emote_type = EMOTE_SOUND | EMOTE_VISIBLE
```

Reply:
I got a little confused when looking at the types we had, this is what i meant, thanks